olga/analytics/views.py

```diff
@@ -54,13 +104,13 @@ def update_students_without_country_value(active_students_amount, students_per_c
 
         return students_per_country
 
-    def create_instance_data(self, received_data, secret_token):  # pylint: disable=too-many-locals
+    def create_instance_data(self, received_data, access_token):  # pylint: disable=too-many-locals
```

Comment:
I am sorry. `pylint: disable=too-many-locals` is a cost from refactor and mind.
I refactored this method some time ago and `pyline disable` not actually relevant for now.